charts/kube-prometheus-stack/templates/exporters/blackbox-exporter/servicemonitor.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.blackboxExporter.enabled }}
```

Comment:
The blackbox-exporter chart already [defines](https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-blackbox-exporter/templates/servicemonitor.yaml) a service monitor. Why is this needed?

Reply:
As does other exporters packaged with kube-prometheus-stack. e.g. [node-exporter](https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-node-exporter/templates/monitor.yaml) 

I followed the existing pattern.

Reply:
The service monitor is already defined in the blackbox exporter chart. It doesn't need to also be added to kube-prometheus-stack.